File: examples/js/basic.mjs
import { ZenosMemoryClient } from '../../sdk/js/zenos-memory-client.mjs';

const memory = new ZenosMemoryClient();

await memory.remember('Zenos Memory JavaScript SDK example ran successfully.', {
  namespace: 'zenos',
  metadata: { source: 'example-js', importance: 4 },
});

const recall = await memory.recall('JavaScript SDK example', { namespace: 'zenos', limit: 3 });
console.log(JSON.stringify({ count: recall.count, first: recall.results?.[0]?.id }, null, 2));

File: examples/python/basic.py
from sdk.python.zenos_memory_client import ZenosMemoryClient

memory = ZenosMemoryClient()

memory.remember(
    'Zenos Memory Python SDK example ran successfully.',
    namespace='zenos',
    metadata={'source': 'example-python', 'importance': 4},
)

recall = memory.recall('Python SDK example', namespace='zenos', limit=3)
print({'count': recall.get('count'), 'first': (recall.get('results') or [{}])[0].get('id')})
